lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -24,7 +24,7 @@
     auth_source=dict(
         type='str',
         choices=['auto', 'cli', 'env', 'credential_file', 'msi'],
-        default='auto'
+        default='auto',
```

Comment:
Although this is a nice change to have, there's no need to make it in this PR. Having it here will significantly increase the number of tests required for the PR to pass CI.

Reply:
@mattclay Trailing comma removed